### PR TITLE
Hello world JS

### PR DESCRIPTION
**Pull request by git-Ball**

### CODE REVIEW

**Review by karastoyanov — CHANGES_REQUESTED** [review]
* Rename the file to `helloWorld-Georgi-Balabanov.js`

* The file should be placed in a `Exercise01-Git/Gergi-Balabanov` directory

* You don't need the `Exercise-demo` dir.

File: Exercise02-Basics/Georgi-Balabanov/BASICS02-014.js
Comment (karastoyanov) [inline Exercise02-Basics/Georgi-Balabanov/BASICS02-014.js:1]: You try to solve this one without using if-else statements, with ternary operators the solution would be something like that:

```js
let result = (letter >= 'A' && letter <= 'Z') ? CHANGE THE UPPERCASE TO LOWERCASE EXPRESSION : (letter >= 'a' && letter <= 'z') ? CHANGE TO LOWERCASE TO UPPERCASE EXPRESSION : letter;
```

File: Exercise03-DataTypes/Georgi-Balabanov/DTYPES01-003.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* DTYPES01-003: Declare two variables. Initialize one of them with your first name and the 
+other one – with your family name. Define a variable called fullRevName with a 
+template string. Use variable substitution for the two simple names, having the family 
+name as a first name. Use comma character as a separator. Print fullRevName on the 
+console.*/
+let firstName ='Georgi';
+let lastName ='Balabanov';
+let fullRevName =`${lastName}.${firstName}`
+console.log(fullRevName)
```
Comment (karastoyanov) [inline Exercise03-DataTypes/Georgi-Balabanov/DTYPES01-003.js:9]: The separator should be `,`

**Review by karastoyanov — CHANGES_REQUESTED** [review]
Small corrections. Also, you may take a look on some of the solutions, where `;` after the print lines, like on [BASIC02-004.js](https://github.com/karastoyanov/JSF-DIW/blob/e73293ffcd536fb628ae728ee0d3625f38c66eb3/Exercise02-Basics/Georgi-Balabanov/BASICS02-004.js)

```js
let randomEmoji1 = '\u{1F703}';
let randomEmoji2 = '\u{1F666}';
let randomEmoji3 = '\u{1F999}';
let randomEmoji4 = '\u{1F333}';
let randomEmoji5 = '\u{1F222}';
console.log(randomEmoji1)
console.log(randomEmoji2)
console.log(randomEmoji3)
console.log(randomEmoji4)
console.log(randomEmoji5)
```

File: Exercise02-Basics/Georgi-Balabanov/BASICS02-014.js
Comment (git-Ball) [reply Exercise02-Basics/Georgi-Balabanov/BASICS02-014.js:1]: Fixed.

File: Exercise03-DataTypes/Georgi-Balabanov/DTYPES01-003.js (hunk 1 above)
Comment (git-Ball) [reply Exercise03-DataTypes/Georgi-Balabanov/DTYPES01-003.js]: Fixed.

**Review by karastoyanov — APPROVED** [review]

**Pull request merged**